music-ecom/src/pages/Home/Home.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import Navbar from "../../Components/Navbar/Navbar";
+import "./Home.css";
+
+// IMAGES IMPORT
+import guitar_wall from "../../Assets/Images/featured__guitar1.jpg";
+import piano from "../../Assets/Images/featured__piano.jpg";
+import drum from "../../Assets/Images/featured__drum.jpg";
+import xylo from "../../Assets/Images/featured__xylo.jpg";
+import brand1 from "../../Assets/Images/new_brand1.jpg";
+import brand2 from "../../Assets/Images/new_brand2.jpg";
+import brand3 from "../../Assets/Images/new_brand3.jpg";
+import brand4 from "../../Assets/Images/new_brand4.jpg";
+
```

Comment:
in your image folder create index.js file and export all the images there and then use it here.

it will reduce the multiline import

Reply:
got it thnks